apiserver/cmd/main.go

```diff
@@ -128,7 +128,7 @@ func startHttpProxy() {
 
 	// Create a top level mux to include both Http gRPC servers and other endpoints like metrics
 	topMux := http.NewServeMux()
-	// Seems /apis (matches /apis/v1alpha1/clusters) works fine
```

Comment:
This change is meaningless without changes in the `.proto` files. I think that apiserver API might not be 'v1' ready. Do you want to make the changes to the API server rest path now ? I think we can move the Kuberay API server paths independent of the underlying kuberay operator clients.

cc: blublinsky

Reply:
To move to v1 we need to modify .proto files, for example:
````
service ClusterService {
  // Creates a new Cluster.
  rpc CreateCluster(CreateClusterRequest) returns (Cluster) {
    option (google.api.http) = {
      post: "/apis/v1alpha2/namespaces/{namespace}/clusters"
      body: "cluster"
    };
  }

  // Finds a specific Cluster by ID.
  rpc GetCluster(GetClusterRequest) returns (Cluster) {
    option (google.api.http) = {
      get: "/apis/v1alpha2/namespaces/{namespace}/clusters/{name}"
    };
  }

  // Finds all Clusters in a given namespace. Supports pagination, and sorting on certain fields.
  rpc ListCluster(ListClustersRequest) returns (ListClustersResponse) {
    option (google.api.http) = {
      get: "/apis/v1alpha2/namespaces/{namespace}/clusters"
    };
  }

  // Finds all Clusters in all namespaces. Supports pagination, and sorting on certain fields.
  rpc ListAllClusters(ListAllClustersRequest) returns (ListAllClustersResponse) {
    option (google.api.http) = {
      get: "/apis/v1alpha2/clusters"
    };
  }

  // Deletes an cluster without deleting the cluster's runs and jobs. To
  // avoid unexpected behaviors, delete an cluster's runs and jobs before
  // deleting the cluster.
  rpc DeleteCluster(DeleteClusterRequest) returns (google.protobuf.Empty) {
    option (google.api.http) = {
      delete: "/apis/v1alpha2/namespaces/{namespace}/clusters/{name}"
    };
  }
}
````
and replace `v1alpha2` with `v1`

We can do it now, but its a breaking change

Reply:
I reverted the change https://github.com/ray-project/kuberay/pull/1482/commits/07b659d71f566288dbfaf7523e42276c63dd4eb6.